.travis.yml

```diff
@@ -18,9 +18,21 @@ install:
   - source activate test-environment
   - python setup.py install
   - pip install --upgrade pytest
+  - pip install visdom torchvision tensorboardX
 
 script:
   - pytest tests/
+  # Smoke tests for examples
+  # mnist.py
+  - python examples/mnist.py --epochs=2
```

Comment:
any reason you went for 2 and not 1? if it's arbitrary than maybe 1 is better so that it ends quicker?

Reply:
Actually no particular reason, I can change to 1